### PR TITLE
DOCSUP-6437: Document the system.opentelemetry_span_log system table

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #14195

### CODE REVIEW

File: docs/en/operations/system-tables/opentelemetry_span_log.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# system.opentelemetry_span_log {#system_tables-opentelemetry_span_log}
+
+Contains information about [trace spans](https://opentracing.io/docs/overview/spans/) for executed queries.
+
+Columns:
+
+-   `trace_id` ([UUID](../../sql-reference/data-types/uuid.md) — ID of the trace for executed query.
```
Comment (adevyatova) [inline docs/en/operations/system-tables/opentelemetry_span_log.md:7]:
```suggestion
-   `trace_id` ([UUID](../../sql-reference/data-types/uuid.md) — The ID of the trace for an executed query.
```

File: docs/en/operations/system-tables/opentelemetry_span_log.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+-   `finish_date` ([Date](../../sql-reference/data-types/date.md)) — The finish date of the `trace span`.
+
+-   `attribute.names` ([Array(String)](../../sql-reference/data-types/array.md)) — [Attribute](https://opentelemetry.io/docs/go/instrumentation/#attributes) names depending on the `trace span`. They are filled in according to the recommendations in the [OpenTelemetry](https://opentelemetry.io/) standard.
+
+-   `attribute.values` ([Array(String)](../../sql-reference/data-types/array.md)) — Attribute values depending on the `trace span`. They are filled in according to the recommendations in the `OpenTelemetry` standard.
+
+**Example**
+
+``` sql
```
Comment (adevyatova) [inline docs/en/operations/system-tables/opentelemetry_span_log.md:27]:
```suggestion
Query:
``` sql
```

File: docs/en/operations/system-tables/opentelemetry_span_log.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+SELECT * FROM system.opentelemetry_span_log LIMIT 1 FORMAT Vertical;
+```
+
+``` text
```
Comment (adevyatova) [inline docs/en/operations/system-tables/opentelemetry_span_log.md:31]:
```suggestion
Result:
``` text
```

**Comment by sevirov** [comment]
@akuzm Please, review this PR.

**Review by akuzm — APPROVED** [review]

**Pull request merged**